superlumic

```diff
@@ -133,10 +130,5 @@ if [ -f "config/$username.yml" ]; then
     setStatusMessage "Running the ansible playbook for $username"
     ansible-playbook -i "localhost," config/$username.yml
 else
-    if [ "travis" = "$username" ]; then
```

Comment:
Could you clarify why this travis code block is being 'removed'?
If I understand the original intent of this code correctly; when running this via Travis CI (username: 'travis'), it would use the 'default' playbook (originally roderik.yml) when doing a test build. Roderik could clarify that.

Personally I worry about that creating a conflict if a user named "travis" wanted to use superlumic. In which case 'travis-ci' might be a better variable name.

In any case, I appreciate your thoughtful contribution and I'd like to hear your thoughts.